tsdb/store_test.go

```diff
@@ -144,6 +145,64 @@ func TestStore_CreateShard(t *testing.T) {
 	}
 }
 
+// Ensure the store can create a new shard.
+func TestStore_StartupShardProgress(t *testing.T) {
+	t.Parallel()
+
+	test := func(index string) {
+		fmt.Println(index)
+		s := MustOpenStore(index)
+		defer s.Close()
+
+		// Create a new shard and verify that it exists.
+		if err := s.CreateShard("db0", "rp0", 1, true); err != nil {
+			t.Fatal(err)
```

Comment:
Use [testify](https://github.com/stretchr/testify). Then this would become `require.NoError(t, CreateShard("db0", "rp0", 1, true))`.

Same other places in here. If you're using a `t.Fatalf`, there's a superior `testify` replacement for it. You do see a lot of codr using `t.Fatalf`, but that's old code that hasn't been updated. New code should use `testify`.

Reply:
Awesome sounds good